frontend/src/components/PasswordChangePage.js

```diff
@@ -0,0 +1,254 @@
+import React, { useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+import {
+  Alert,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  TextField
+} from "@mui/material";
+import logo from '../assets/logo.png';
+import homePageImage from '../assets/homePageImage.svg'
+import {URL_USER_SVC} from "../configs";
+import {STATUS_CODE_OK, STATUS_CODE_BAD_REQUEST} from "../constants";
+
+const useStyles = createUseStyles({
+  homePage: {
+    height: "100vh",
+    backgroundColor: "#b5dce9",
+  },
+  banner: {
+    marginBottom: "20px",
+    width: "100%",
+    height:"fit-content",
+    backgroundColor: "white",
+    justifyContent: "space-between",
+    display: "flex",
+    boxShadow: "rgba(0, 0, 0, 0.12) 0px 1px 3px, rgba(0, 0, 0, 0.24) 0px 1px 2px"
+  },
+  logo: {
+    margin: "20px 0 20px 20px",
+    height: "35px",
+    width: "auto",
+  },
+  logoutButton: {
+    margin: "20px 20px 20px 0",
+  },
+  pageBody: {
+    display: "flex",
+    flexDirection:"column",
+    width: "95%",
+    margin: "auto",
+  },
+  passwordChangeCardContainer: {
+    display: "flex",
+    margin: "auto",
+    height: "fit-content",
+    backgroundColor: "white",
+    boxShadow: "rgba(0, 0, 0, 0.12) 0px 1px 3px, rgba(0, 0, 0, 0.24) 0px 1px 2px",
+    borderRadius: "10px"
+  },
+  passwordChangeCard: {
+    padding: "40px",
+    display: "block",
+    margin: "auto",
+  },
+  backButton: {
+    border: "none",
+    backgroundColor: "transparent",
+    fontSize: "16px",
+    fontStyle: "italic",
+    color: "#898989",
+    cursor: "pointer",
+    marginBottom: "20px",
+  },
+  changePasswordButton: {
+    height: "45px",
+    margin: "10px auto",
+    textTransform: "Capitalize",
+    fontWeight: "bold"
```

Comment:
```suggestion
    fontWeight: "bold",
    float: "right"
```
Perhaps an alignment of the button to the right? Left looks a little weird.
<img width="709" alt="image" src="https://user-images.githubusercontent.com/61898455/196642990-ed94242c-ab63-4b39-a15b-76738969ed45.png">